src/Kestrel.Core/Internal/Http2/Http2Connection.cs

```diff
@@ -141,6 +141,12 @@ public void Stop()
                 error = ex;
                 errorCode = ex.ErrorCode;
             }
+            catch (HPackDecodingException ex)
```

Comment:
Does every HPackDecodingException that gets thrown get caught here? If so, you might as well log all the new exception messages. Should be easy, right?

Reply:
There are TODOs for logs in a bunch of places. I'm going to address those as a separate change (need to define log messages for all the possible connection errors, etc.).